test/test-network/systemd-networkd-tests.py

```diff
@@ -3027,8 +3025,20 @@ def test_dhcp_client_ipv4_only(self):
         self.assertRegex(output, r'192.168.5.7 proto dhcp scope link src 192.168.5.181 metric 1024')
         self.assertRegex(output, r'192.168.5.8 proto dhcp scope link src 192.168.5.181 metric 1024')
 
-    def test_dhcp_client_ipv4_use_routes_no(self):
-        copy_unit_to_networkd_unit_path('25-veth.netdev', 'dhcp-server-veth-peer.network', 'dhcp-client-ipv4-use-routes-no.network')
+    def test_dhcp_client_ipv4_use_routes_gateway(self):
+        for (routes, gateway) in itertools.product([True, False, None], repeat=2):
+            self.setUp()
+            with self.subTest(routes=routes, gateway=gateway):
+                self._test_dhcp_client_ipv4_use_routes_gateway(routes, gateway)
+            self.tearDown()
+
+    def _test_dhcp_client_ipv4_use_routes_gateway(self, routes, gateway):
+        testunits = ['25-veth.netdev', 'dhcp-server-veth-peer.network', 'dhcp-client-ipv4-use-routes-use-gateway.network']
```

Comment:
Please wrap the long lines.

Reply:
ack, I shortened this line and a couple others down, but some are still over 109 chars; there are lots of other 109+ char lines in this test file, so I wasn't sure if it was a hard limit in the test files, but I can do more wrapping if preferred.

Reply:
There is no hard rule. But in this case each-item-on-a-line is easier to read (and later diffs look much better too).